go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.1.5
+	github.com/alecthomas/kingpin v2.2.6+incompatible
```

Comment:
I see this was not needed as this library is already imported as `kingpin "gopkg.in/alecthomas/kingpin.v2"`

https://github.com/projectcontour/contour/blob/0d684bc09c10668fc7d139dc5e25919d51498b1f/cmd/contour/contour.go#L24

Will make the changes once other code is reviewed.

Reply:
Done.